unstructured/partition/pdf.py

```diff
@@ -94,13 +97,28 @@ def partition_pdf(
         The last modified date for the document.
     """
     exactly_one(filename=filename, file=file)
+
+    if languages and ocr_languages:
```

Comment:
this seems strange to raise, since we have a default value for languages. might be confusing to users if they didn't actually pass in the `languages` param

Reply:
was following this pattern https://github.com/Unstructured-IO/unstructured/blob/e4e25c9febe8e46874cbb91f0201cfaf5330a672/unstructured/partition/auto.py#L185

Reply:
Yes I think this makes sense to raise when both params are passed in by the user as in the example you linked. But in this case where we're defining a default we will return this error message if they only pass in `ocr_languages` param which is confusing.